Add note to docs on how to override privileges (#9792)
minetest/minetest: doc/lua_api.txt

      by default.
    * To allow players with `basic_privs` to grant, see the `basic_privs`
      minetest.conf setting.

>     * To modify an already registered privilege just edit the value found in minetest.registered_privileges

Reviewer: ```suggestion
    * Registered privileges can be modified directly in
      `minetest.registered_privileges`.
```

Reviewer 2: Please also apply this wording in other places, it's better.

    * Map of registered tool definitions, indexed by name
* `minetest.registered_entities`
    * Map of registered entity prototypes, indexed by name

>     * You can use this to change the values of already registered entities

Reviewer: ```suggestion
    * Values in this table may be modified directly.
      Note: the new values will only affect newly spawned entities.
```

    * Map of registered chat command definitions, indexed by name
* `minetest.registered_privileges`
    * Map of registered privilege definitions, indexed by name

>     * You can use this to change the values of already registered privileges

Reviewer: How to use this? Superfluous `already`.

```suggestion
    * Registered privileges can be modified directly in this table.
```

    * Map of registered tool definitions, indexed by name
* `minetest.registered_entities`
    * Map of registered entity prototypes, indexed by name
    * Values in this table may be modified directly.

>       Note: the new values will only affect entities spawned afterwards

Reviewer 3: This is not true - it uses metatables, so the entity table will immediately change for all entities. However, initial properties are only read when entities are spawned

```suggestion
      Note: changes to initial properties will only affect entities spawned afterwards,
      as they are only read when spawning.
```
